tests/test_fetch_topic.py

```diff
@@ -0,0 +1,26 @@
+import pytest
```

Comment:
you don't seem to be using most of the imports you have here.

Reply:
Removed.